src/components/EntityScorecardsCard.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from "react";
+import { InfoCard } from "@backstage/core-components";
+import { useEntity } from "@backstage/plugin-catalog-react";
+import useAsync from "react-use/lib/useAsync";
+import { useApi } from "@backstage/core-plugin-api";
+import { Progress, ResponseErrorPanel } from "@backstage/core-components";
+import Box from "@material-ui/core/Box";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+
+import { dxApiRef, Scorecard, ScorecardCheck } from "../api";
+import { BrandedCardTitle } from "./BrandedCardTitle";
+import { CheckResultBadge } from "./CheckResultBadge";
+import { COLORS } from "../styles";
+
+const DEFAULT_NO_LEVEL_COLOR = "#CBD5E1";
+
+type EntityScorecardsCardProps = {
+  contentMaxHeight?: string | number;
+};
+
+export function EntityScorecardsCard({
+  contentMaxHeight = "20rem",
+}: EntityScorecardsCardProps) {
+  const dxApi = useApi(dxApiRef);
+
+  const { entity } = useEntity();
+
+  const entityIdentifier = entity.metadata.name;
+
+  const [tab, setTab] = useState<"levels" | "checks">("levels");
+
+  const {
+    value: response,
+    loading,
+    error,
+  } = useAsync(() => {
+    return dxApi.scorecards(entityIdentifier);
+  }, [dxApi, entityIdentifier]);
+
+  if (loading) {
+    return <Progress />;
+  }
+
+  if (error) {
+    return <ResponseErrorPanel error={error} />;
+  }
+
+  if (!response) {
+    throw new Error("Unreachable");
+  }
+
+  const scorecards = response.scorecards;
+
+  const flattenedChecks = scorecards.flatMap((scorecard) => scorecard.checks);
+
+  return (
+    <InfoCard
+      title={
+        <Box
+          sx={{
+            display: "flex",
+            flexDirection: "column",
+            gridGap: 16,
+            marginBottom: -20,
+          }}
```

Comment:
There are some more custom styles in this card than others so I think we want to go with this `<Box sx={{ ... }}>` strategy for styling them. Thoughts?

Reply:
Yeah I like it. I tend to use the "box" components from these type of libraries more often since it matches what we do with tailwind.